src/heuristics.jl

```diff
@@ -2,3 +2,78 @@
 
 ## Have a general interface for heuristics 
 ## so that the user can link in custom heuristics.
+"""
+    Boscia Heuristic
+
+Interface for heuristics in Boscia.    
+`h` is the heuristic function receiving as input the tree, the bounded LMO and a point x (the current node solution).
+It returns the heuristic solution (can be nothing, we check for that) and whether feasibility still has to be check.
+`prob` is the probability with which it will be called.        
+"""
+# Would 'Heuristic' also suffice? Or might we run into Identifer conflicts with other packages?
+struct Heuristic
+    h::Function
+    prob::Float64
+    identifer::Symbol
+end
```

Comment:
```suggestion
struct Heuristic{F}
    h::F
    prob::Float64
    identifer::Symbol
end
```
to have it type-stable

Reply:
also, maybe `run_heuristic` instead of `h`